docs/content/concepts/layer/control/_index.md

```diff
@@ -21,10 +21,12 @@ The **[Catalog Model](catalog/)** provides a structured, machine-readable repres
   
 It is important to note that the OSCAL catalog model is not a catalog document format, since the introductory prose included in many control catalogs is not present (or supported) in the OSCAL catalog model. Instead, the OSCAL catalog formats provide a robust syntax for representing collections of controls, including control statements, assessment objectives, and other control details. This structured form of a control catalog allows control information to be easily imported, exported, indexed, and searched by applications.
 
+Controls must also be encoded in a standard machine-readable form. The OSCAL model gives the designers of catalogs great flexibility in the details of how controls are defined, with their constituent parts. This is described further in a page providing a high-level introductory walkthrough of an [Annotated Example](catalog/sp800-53rev5-example) of a control, as defined by NIST SP 800-53 Rev 5.
+
 ## Profile Model
 
-The **[Profile Model](profile/)** provides a structured, machine-readable representation of a **[baseline](../../terminology/#baseline)**. An OSCAL profile defines a specific set of selected security control requirements from one or more control catalogs, or by customizing other profiles.
+The **[Profile Model](profile/)** provides a structured, machine-readable representation of a **[baseline](../../terminology/#baseline)**. As the starting point for defining an organization's security mission and security posture, a baseline must be defined by any organization undertaking an RMF-based (Risk Management Framework) security program or security assessment, typically by selecting and adapting controls from an authoritative catalog of controls, or from another baseline that has already been defined and described.
```

Comment:
It seems here we are also implying the RMF semantic model is _the_ model for OSCAL information. Is that intentional and something we want to do in this segment?

Reply:
Are baselines (selections of control sets from catalog) part of non-RMF-based security programs or assessments? I could be educated.

Reply:
Some might confuse RMF with NIST RMF. How about the following to be more clear?
```suggestion
The **[Profile Model](profile/)** provides a structured, machine-readable representation of a **[baseline](../../terminology/#baseline)**. As the starting point for defining an organization's security mission and security posture, a baseline must be defined by any organization undertaking a risk-based security program or security assessment, typically by selecting and adapting controls from an authoritative catalog of controls, or from another baseline that has already been defined and described.
```